src/uikit.cs

```diff
@@ -2510,10 +2511,12 @@ interface UIApplication {
 
 		[iOS (10,0), TV (10,0)]
 		[Export ("openURL:options:completionHandler:")]
+		[Async]
```

Comment:
I am a little worried about this one since this could open another App (be backgrounded) to do stuff and then **maybe** come back to our app, can our Async handle this scenario?